NMS-15274: Move severities chart (#5761)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/bsm/operational-status.adoc

= Operational Status

Every business service maintains an operational status that represents its overall status.
The overall status is calculated from the edges, using the map and reduce functions, and uses the <<deep-dive/events/event-configuration.adoc#severities, severities>> from associated events and alarms.

If a business service changes its operational status, a `uei.opennms.org/bsm/serviceOperationalStatusChanged` event is generated.

> In cases where the operational status changed from `Normal` to a higher severity, a `uei.opennms.org/bsm/serviceProblem` event is generated and is assigned the same severity as the business service.

Reviewer: ```suggestion
In cases where the operational status changes severity to a state other than `Normal`, a `uei.opennms.org/bsm/serviceProblem` event is generated and is assigned the same severity as the business service.
```

When the business service returns to `Normal` severity, a `uei.opennms.org/bsm/serviceProblemResolved` event is generated.

> NOTE: Use the `serviceProblem` and `serviceProblemResolved` events for notifications or ticketing integration.

Reviewer: ```suggestion
TIP: You can use the `serviceProblem` and `serviceProblemResolved` events for notifications or ticketing integration.
```
